SocketHttpListener/Net/HttpListenerRequest.cs

```diff
@@ -88,7 +86,7 @@ public Encoding ContentEncoding
                         }
                     }
                 }
-                return TextEncodingExtensions.GetDefaultEncoding();
+                return Encoding.UTF8;
```

Comment:
I am slightly weary of this change, can we be sure that default encoding is always utf-8?